prod-refractr.yml

```diff
@@ -1040,7 +1042,7 @@ refracts:
 # SE-2418
 - dsts:
     - redirect: webthings.io/
-  srcs: 
```

Comment:
my editor auto-cleans trailing whitespace. I can remove these from the PR if you'd prefer.